client/clothing.lua

```diff
@@ -797,13 +797,9 @@ end
 
 RegisterNetEvent('qb-radialmenu:ResetClothing', ResetClothing)
 
-function ToggleClothing(whic, extra)
-	local which = whic
-	if type(whic) == "table" then
-		which = tostring(whic.id)
-	end
-	Wait(50)
-
+function ToggleClothing(data)
+	local which = data.id
+	local extra = data.extra

```

Comment:
The indentation here is 1 layer too deep

Reply:
should be good now ? github being weird i guess..